rockhound/sample_data.py

```diff
@@ -0,0 +1,201 @@
+"""
+Fetch sample datasets
+"""
+import pandas as pd
+import xarray as xr
+from pooch import Decompress
+
+from .registry import REGISTRY
+
+
+def fetch_baja_bathymetry():
+    """
+    Fetch sample bathymetry data from Baja California.
+
+    This is the ``@tut_ship.xyz`` sample data from the `GMT
+    <http://gmt.soest.hawaii.edu/>`__ tutorial.
+
+    Returns
+    -------
+    data : :class:`pandas.DataFrame`
+        The bathymetry data. Columns are longitude, latitude, and bathymetry
+        (in meters) for each data point.
+
+    See also
+    --------
+    setup_baja_bathymetry_map: Utility function to help setup a Cartopy map.
```

Comment:
These should probably be removed since we're not porting the plotting functions here.